.github/workflows/marin-unit-tests.yaml

```diff
@@ -57,7 +57,7 @@ jobs:
           python -m pip install --upgrade pip
           pip install "uv>=0.7.19" toml
           npm install -g pandiff
-          uv sync --package marin --extra cpu --group test --frozen
+          uv sync --package marin --extra cpu --extra=quality_dedup_consolidate --group test --frozen
```

Comment:
nit: document this somewhere? Though IDK where the best place for it is, perhaps [lib/dupekit/README.md](https://github.com/marin-community/marin/blob/main/lib/dupekit/README.md)?

Reply:
Good point - updated in e92dfd9